### PR TITLE
MM-32431 - Fix local e2e tests

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Final e2e fix for awhile (I hope)
- Some tests were failing locally, figured out why, here are the fixes.

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-32431

### CODE REVIEW

File: server/command/command.go [hunk 1]
```diff
@@ -913,7 +913,7 @@ func (r *Runner) actionTestCreate(params []string) {
 		return
 	}
 
-	creationTimestamp, err := time.Parse("2006-01-02", params[1])
+	creationTimestamp, err := time.ParseInLocation("2006-01-02", params[1], time.Now().Location())
```
Comment (agarciamontoro) [inline server/command/command.go:916]: Oh, great catch!

File: tests-e2e/cypress/integration/backstage/incident_list_spec.js [hunk 2]
```diff
@@ -13,10 +13,25 @@ import {TINY} from '../../fixtures/timeouts';
...
+    let newTeamName;
+    let newTeamId;
     let userId;
     let playbookId;
 
     before(() => {
+        // # Login as the sysadmin
+        cy.apiLogin('sysadmin');
+
+        // # Create a new team for the welcome page test
```
Comment (agarciamontoro) [inline tests-e2e/cypress/integration/backstage/incident_list_spec.js:25]: So... do the tests start with data from previous CI jobs?

**Review by lieut-data — APPROVED** [review]
Magic.

File: tests-e2e/cypress/support/ui_commands.js [hunk 3]
```diff
@@ -194,7 +194,11 @@ Cypress.Commands.add('editPost', (postId, newMessage) => {
  * @param {String} channelName - Display name of the channel.
  */
 Cypress.Commands.add('uiSwitchChannel', (channelName) => {
-    cy.get('body').type('{ctrl}k');
+    if (Cypress.platform === 'darwin') {
+        cy.get('body').type('{cmd}k');
+    } else {
+        cy.get('body').type('{ctrl}k');
+    }
```
Comment (agarciamontoro) [inline tests-e2e/cypress/support/ui_commands.js:201]: Cool!

**Review by agarciamontoro — APPROVED** [review]
Yay! :tada:  Thank you for all your work trying to preserve the E2E tests :pray:

File: tests-e2e/cypress/integration/backstage/incident_list_spec.js (hunk 2 above)
Comment (cpoile) [reply tests-e2e/cypress/integration/backstage/incident_list_spec.js:25]: @agarciamontoro They do if you run them on your local machine more than once. :)

Comment (agarciamontoro) [reply tests-e2e/cypress/integration/backstage/incident_list_spec.js:25]: Oh yeah, of course... Thanks!

**Pull request merged**